src/SDKs/RecoveryServices/Management.RecoveryServices/Microsoft.Azure.Management.RecoveryServices.csproj

```diff
@@ -11,6 +11,9 @@
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Update="Microsoft.Rest.ClientRuntime" Version="2.3.10" />
```

Comment:
@sonathan this is not needed.
We control the common packages when we have tested and ready to move to a new version of clientruntime throughout the repo.
So when the new version is out and ready to be consumed, this change will be done for the entire repo.
So please revert this change.
If we allow this change, controlling ClientRuntime version across repo will be fragmented and not manageable.

Reply:
@Shahabhijeet Thank-you for letting me know of this. I'll update this in next commit.